packages/network-controller/src/create-auto-managed-network-client.ts

```diff
@@ -89,9 +95,29 @@ export function createAutoManagedNetworkClient<
     rpcEndpointUrl: string,
   ) => Omit<PollingBlockTrackerOptions, 'provider'>;
   messenger: NetworkControllerMessenger;
+  isRpcFailoverEnabled: boolean;
 }): AutoManagedNetworkClient<Configuration> {
+  let isRpcFailoverEnabled = givenIsRpcFailoverEnabled;
   let networkClient: NetworkClient | undefined;
 
+  const ensureNetworkClientCreated = (): NetworkClient => {
+    networkClient ??= createNetworkClient({
+      configuration: networkClientConfiguration,
+      getRpcServiceOptions,
+      getBlockTrackerOptions,
+      messenger,
+      isRpcFailoverEnabled,
+    });
+
+    if (networkClient === undefined) {
+      throw new Error(
+        "It looks like `createNetworkClient` didn't return anything. Perhaps it's being mocked?",
```

Comment:
Do we have any spec on standardized error message formatting?

Reply:
I don't believe so, no. Is there another format you recommend?